fortanix-vme/eif-tools/src/bin/eif-sign.rs

```diff
@@ -0,0 +1,92 @@
+use std::fs::File;
+use std::io::Write;
+use std::path::Path;
+use std::process::exit;
+
+use clap::{Arg, crate_authors, crate_version};
+use env_logger;
+use log::LevelFilter;
+use nitro_cli::EifBuilder;
+use sha2;
+use sha2::Digest;
+
+use eif_tools::*;
+
+fn main() {
+    let args = clap::App::new("eif-sign")
+        .author(crate_authors!())
+        .about("Sign an EIF file")
+        .version(crate_version!())
+        .arg(Arg::with_name("verbose")
+            .short("v")
+            .long("verbose")
+            .help("Print extra information about the signing process"))
+        .arg(Arg::with_name("elffile")
```

Comment:
You have the same typo here: `eiffile` instead of `elffile`

Reply:
fixed, thanks!

Reply:
The "eif" name is so annoying. It's basically typo squatting on developers.